pkg/morph/client/client.go

```diff
@@ -419,6 +420,30 @@ func (c *Client) IsValidScript(script []byte, signers []transaction.Signer) (res
 	return result.State == vmstate.Halt.String(), nil
 }
 
+// VotesFor returns a key the provided account has voted with its NEO
+// tokens for. Nil key with no error it returned if the account has no NEO.
+func (c *Client) VotesFor(addr util.Uint160) (*keys.PublicKey, error) {
```

Comment:
`Votes` sound plural here, while we only have a single vote for a single account. Maybe `AccountVote`.

Reply:
a meant "`addr` votes for". was it hard to understand?

Reply:
I see what was meant, but it doesn't sound right anyway. It's not `addr.VotesFor()` also, it's `client.VotesFor`, but client doesn't vote. In general functions are verbs, so some `GetAccountVote()` is more appropriate. At the same time we often omit `get` for data accesses.

Reply:
ok, `AccountVote`